jasmine-es6-promise-matchers.js

```diff
@@ -40,7 +40,16 @@
     RESOLVED: 'resolved',
   };
 
+  function isError(value) {
+    return typeof value === 'object' && value instanceof Error;
+  }
+
   var verifyData = function(actualData, expectedData, done) {
+    if (isError(actualData) && isError(expectedData)) {
```

Comment:
This comparison will also be run for _resolved_ promises, but I guess that's okay.

Reply:
Yeah, I was trying to figure how to avoid that, or how to also write something like `.not.toBeRejectedWith(new Error('something'))`...  if you want to actually test if a particular error is not thrown, but that lead to more complexity.

Maybe in another PR later?

Reply:
@bvaughn OK I figured how to make it work with both positive and negative assertion.
Slightly bigger PR tho